Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs

```diff
@@ -99,6 +99,76 @@ public IEnumerator Test02_ManipulationHandlerGazeHover()
             yield return null;
         }
 
+        /// <summary>
+        /// Test creates an object with manipulationhandler verifies translation with articulated hand
+        /// as well as forcefully ending of the manipulation
+        /// from gaze provider.
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator ManipulationHandlerForceRelease()
+        {
+            TestUtilities.InitializeMixedRealityToolkitAndCreateScenes(true);
```

Comment:
I believe this happens in SetupMrtk already, these two lines should not be needed.